Inc/Utils/control_util.h

```diff
@@ -29,8 +29,8 @@ typedef struct {
 	float timeDiff;
 }PIDvariables;
 
-
-float PID(float err, PIDvariables K){
+//PID control, static to not have multiple implementation error
+static float PID(float err, PIDvariables K){
 	static float I = 0;
```

Comment:
I needs to be saved in the struct, because it is not shared between different PID loops

Reply:
also make the PID variables a pointer, otherwise it will go out of scope

Reply:
How do you mean?